src/components/Shop/BasketWindow/PurchaseButton.tsx

```diff
@@ -1,13 +1,30 @@
-import React, { FC } from "react";
+import React, { useContext, FC } from "react";
```

Comment:
I don't think this component needs to exist when we have the Button component. I think all the logic can be moved into the BasketWindow component.